### DISCUSSION TITLE
Adding "*" asterisk for significant results

### DISCUSSION THREAD

**Discussion by follhim** [rempsyc/rempsyc#12] [Q&A] [answered]
I noticed that in the examples and guides, you added in the notes "\*" for significant results. 
Is there any way that you can automate a "\*" or "**" based on the p-value within nice_table?

**Comment by rempsyc** [comment]
That's a very reasonable request! Even though I don't use this personally, it is a very commonly used stylistic customization, so I think it makes a lot of sense to add it as an option for those who would like to do that.

**Reply by follhim** [reply]
I also just want to say how much I love this package and function so much! Thank you so much for your work and contribution!

**Comment by rempsyc** [comment, marked as answer]
Thanks for your feedback @follhim! In version `0.1.1.3`, significance stars are now the default. You can see the result here: https://rempsyc.remi-theriault.com/articles/table

Let me know if that satisfies your needs!
